azerothcore/azerothcore-wotlk
# fix(DB/Creatures): Removed interrupt mechanic immune mask for Befoule…

…d Water Elemental.

Fixes #7110



## Issues Addressed:

- Closes #7110
- Closes https://github.com/chromiecraft/chromiecraft/issues/1258

## Tests Performed:

- Not tested

## How to Test the Changes:

1. Have a paladin with learnt Hammer of Justice spell
2. `.go creature id 3917`
3. Engage combat with mob
4. Wait for the mob to start casting its interruptable spell
5. While mob is still casting use HoJ spell on it
6. Verify if mob's cast is not proceeding


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
